NumLib/Fem/Integration/IntegrationPoint.h

```diff
@@ -0,0 +1,88 @@
+/**
+ * \copyright
```

Comment:
Point wise Neumann BC dose not need integration, the value of the BC can be input directly.